.github/workflows/push_release.yml

```diff
@@ -34,11 +34,36 @@ jobs:
           ${{ runner.os }}-pip-
     - name: Install dependencies
       run: make install
-    
+
+    - name: Autobump version
+      run: |
+        # from refs/tags/v1.2.3 get 1.2.3
+        VERSION=$(echo $GITHUB_REF | sed 's#.*/v##')
+        echo $VERSION
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+        poetry run bump2version release --verbose --commit --no-tag --new-version $VERSION
```

Comment:
IIRC, I added targets to the makefile for bumping. Are any of those sufficient? I was trying to get it to the point where we could do anything we want through make so we wouldn't have to copy/paste commands out of the CI files to reproduce them locally.